tests/Tests/500__jupyterhub/test-jupyterlab-notebook.robot

```diff
@@ -54,3 +54,28 @@ Can Launch Python3 Smoke Test Notebook
   #Get the text of the last output cell
   ${output} =  Get Text  (//div[contains(@class,"jp-OutputArea-output")])[last()]
   Should Not Match  ${output}  ERROR*
+
+Verify A Default Image Is Provided And Starts Successfully
+    [Documentation]    Verify that, if a user doesn't explicitly select any jupyter image
+    ...    a default one is selected and it can be spawned successfully
+    [Tags]    Sanity
+    ...       ODS-469
+    Launch JupyterHub Spawner From Dashboard
+    Spawn Notebook
+    ${has_spawn_failed} =    Has Spawn Failed
+    Should Be Equal As Strings    ${has_spawn_failed}    False
+    Wait For JupyterLab Splash Screen  timeout=30
+    Open With JupyterLab Menu  File  New  Notebook
+    Sleep    1
+    Maybe Close Popup
+    Verify Notebook Name And Image Tag
+
+
+*** Keywords ***
+Verify Notebook Name And Image Tag
+    [Documentation]    Verifies that expected notebook is spawned and image tag is not latest
+    @{user_data} =    Get Previously Selected Notebook Image Details
```

Comment:
I think the previosly selected image should be fetched before spawning a new jupyter server, otherwise the check is meaningless. I would ask @lugi0 to have a look too

Reply:
Updated